### ISSUE TITLE
Loot procedure on attacker side

### ISSUE THREAD

**Issue by cherrywoods** [cherrywoods/wirvsvirus-gamification-toiletpaper-clans#11]

**Comment by karlgrossmann** [comment]
Die Modals gibt's quasi schon, ich passe die heute oder morgen nochmal an und implementiere sie Frontend-seitig :)

**Comment by karlgrossmann** [comment]
Die UI vom Mockup ist implementiert 👍 Close einfach das Issue, wenn es das war, dass du mit der Loot procedure meintest :)

**Comment by cherrywoods** [comment]
ja, dass meinte ich auch. Ist aber auch die ganze benachrichtigungs mechanik implemenentiert?

**Comment by cherrywoods** [comment]
also dass mit alle team member müssen auf einen push request antworten oder so
